.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     if: (github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository)
     strategy:
       matrix:
-        go: [ '1.20', '1.21', '1.22' ]
+        go: [ '1.20', '1.21', '1.22', '1.23' ]
```

Comment:
`1.20` is not needed in fact. We are ok if we maintain 2 versions, 3 is very nice.

Reply:
`go.mod` says `1.20` so `1.20` needs testing. nothing on this commit requires dropping `1.20`. if we want `1.20` dropped for `v0.15` I believe that should be a different PR